client/src/pages/spectrogram/SettingsPane.tsx

```diff
@@ -154,7 +158,7 @@ print("Time elapsed:", (time.time() - start_t)*1e3, "ms")`,
           className="range range-xs range-primary"
           value={state.zoomLevel}
           min={1}
-          max={10}
+          max={zoomLevels.length}
```

Comment:
Isn't having the minimum here being 1 would would not allow to select the first one in the array and the max you go over the max index of the array